app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
+require "#{Rails.root}/lib/helpers/api_helpers"
+
 class ApplicationController < ActionController::API
+  include APIHelpers
```

Comment:
My intention here is to have some helper methods that are available to all the controllers (in this case, it's a method that generates a list of acceptable upsert parameters based on the model). I know there are a few Rails-y patterns to accomplish this goal, so I'm open to feedback here.